src/jsonrpc/miner_jsonrpc_txn.erl

```diff
@@ -31,7 +31,7 @@ handle_rpc(<<"txn_queue">>, []) ->
 handle_rpc(<<"txn_add_gateway">>, #{ <<"owner">> := OwnerB58 } = Params) ->
     try
         Payer = optional_binary_to_list(
-            maps:get(payer, Params, undefined)
+            maps:get(<<"payer">>, Params, undefined)
         ),
         Owner = binary_to_list(OwnerB58),
```

Comment:
Is there some variable naming issue here? `OwnerB58`, from the name atleast, would be a string already so `binary_to_list(on_a_string)` would fail no?

Unless `OwnerB58` is in fact an `OwnerPubkeyBin` in which case it should just do `libp2p_crypto:bin_to_b58(OwnerPubkeyBin)` to get a string.

Reply:
It's JSON RPC. Inbound "strings" come in as `binary()`, thus OwnerB58 will be a binary. Payer, likewise, will be a binary. Perhaps you'd feel more comfortable if the variable at line 33 would change names from `Payer` to `PayerB58`?